app/queries/reasons_for_rejection_count_query.rb

```diff
@@ -24,7 +24,7 @@ def reason_counts
     rows = ActiveRecord::Base.connection.exec_query(
       reason_counts_sql,
       'SQL',
-      [[nil, Time.zone.now.beginning_of_month]],
```

Comment:
This was removed in https://github.com/rails/rails/commit/2d821ef0c5356d860eba2d2341b4db71b67f6594
From what I can see it preserves the previous behaviour, however worth double checking with @stevehook

Reply:
Looks good to me - that param was removed so redundant.